pallets/clusters/src/lib.rs

```diff
@@ -233,13 +234,22 @@ pub mod pallet {
 				minimum_balance,
 			)?;
 
-			// create a pure proxy
-			pallet_proxy::Pallet::<T>::create_pure(
-				origin.clone(),
-				T::ProxyType::default(),
-				T::BlockNumber::zero(),
-				0,
-			)?;
+			// create a proxy
+			let proxy_def = pallet_proxy::ProxyDefinition {
```

Comment:
I mean if create_pure did the same thing I'm fine to use create_pure.... btw. Don't misunderstand my comment.

Reply:
There is a detail that make it different, and it's the reason why I ended up copying from pallet_account. It is in `pallet_proxy::Proxies::<T>::insert`.
The `create_pure` function internally creates a pure account that `insert `, while we are actually doing `insert `of an account that we create (in this case the `vault `- i.e. the account of the cluster).

Reply:
nvm I see we do `reseve()` later. Could we change `>` to `≥` then 😅 ?